charts/dapr/charts/dapr_sidecar_injector/values.yaml

```diff
@@ -37,6 +37,8 @@ kubeClusterDomain: cluster.local
 ignoreEntrypointTolerations: "[{\\\"effect\\\":\\\"NoSchedule\\\",\\\"key\\\":\\\"alibabacloud.com/eci\\\"},{\\\"effect\\\":\\\"NoSchedule\\\",\\\"key\\\":\\\"azure.com/aci\\\"},{\\\"effect\\\":\\\"NoSchedule\\\",\\\"key\\\":\\\"aws\\\"},{\\\"effect\\\":\\\"NoSchedule\\\",\\\"key\\\":\\\"huawei.com/cci\\\"}]"
 hostNetwork: false
 healthzPort: 8080
+objectSelector: ~
```

Comment:
Can you replace `~` with an empty object? Just to keep it consistent with other default values.
Also, it would be good to provide examples of both `namespaceSelector` and `objectSelector`.